build.sbt

```diff
@@ -33,6 +33,10 @@ lazy val root = (project in file("."))
       "ch.qos.logback" % "logback-classic" % "1.2.3",
       "org.scalatest" %% "scalatest" % "3.2.2" % Test
     ),
+    javaOptions in Universal ++= Seq(
+      s"-Dpidfile.path=/dev/null",
```

Comment:
Why is the pidfile path set to /dev/null? I've seen the effects of replacing /dev/null with a real file (they're not fun!) - I'm sure that's not the intention/what will happen here 😄